src/main/java/org/nprentza/DroolsAgentApp.java

```diff
@@ -28,22 +27,42 @@ public class DroolsAgentApp {
     private List<Integer> deny;
 
     private String DRL = DrlConverter.preamble()
-            + DrlConverter.rule("AllowAdmin", "role", "admin", "allow")
-            + DrlConverter.rule("DenyGuest", "role", "guest", "deny")
+            + DrlConverter.rule("AllowAdmin", "role", "==", "admin", "allow")
+            + DrlConverter.rule("DenyGuest", "role", "==", "guest", "deny")
+            // Uncomment this rule to create an upper bound for the gap:
+            // + DrlConverter.rule("DenyDead", "age", ">",200, "deny")
```

Comment:
@nprentza Uncomment this as a step 2 and then change the testing predictor's age from 50 to 500 in `Main.java`.